src/RestAPI/dlwsrestapi.py

```diff
@@ -1078,7 +1089,7 @@ def endpoint_exist(endpoint_id):
         if "ssh" in requested_endpoints:
             # setup ssh for each pod
             for pod_name in pod_names:
-                endpoint_id = "endpoint-" + pod_name + "-ssh"
+                endpoint_id = "e-" + pod_name + "-ssh"
```

Comment:
Why do we want to do this? This is not as descriptive as "endpoint".

Reply:
To reduce the length of `id`, k8s have a limit of 63 characters.